README.md

```diff
@@ -101,14 +109,14 @@ To do this, run the following:
 workon cfgov-refresh
 
 # cd into the /dist/ directory.
-cd ./dist
+cd cfgov/v1/jinja2/v1
```

Comment:
@anselmbradford yea we're i'm just moving the location of the processed templates to cfgov/v1/jinja2/v1 in preparation for hooking wagtail into refresh